README.md

```diff
@@ -57,11 +57,13 @@ the world.
 
 ## Mailing Lists
```

Comment:
Can you replace this header with something like "Getting in touch"/"Contact"/...

Reply:
I used "Getting in touch"